docs/comment_copy_guidelines.md

```diff
@@ -0,0 +1,127 @@
+# Comment Copy Guidelines
+
+This document sets guidelines for the comments produced by the analyzer. The
+[Feedback guidlines][git-feedback-guidelines] talk about _when_ to comment and
+_what_ to comment. This document talks mostly about _how_ to comment.
+
+## Working directory for contents
+
+Comments for the analyzer live in the [`exercism/website-copy`][git-website-copy]
+repository. Changes should be made via a PR on this repository, changing only
+the comments for one analyzer at a time. These will get the
+[`type/analyzer-comments` label][git-website-copy-label].
+
+### Generic comments
+
+Some comments are language specific and not exercise specific. These should be
+extracted into a generic comments file, in order to not duplicate commentary,
+which would lead to either outdated commentary or shotgun surgery when a change
+is made.
+
+## CI
+
+Because the comments don't live in the same repository as the analyzer, each
+analyzer should have CI that chacks if the comments used in that specific
+analyzer (those that can become output), are comments on the `master` branch, on 
+the  [`exercism/website-copy`][git-website-copy] repository.
+
+At moment of writing, [this issue][issue-ci-comments] tracks the status of any
+generalisation of this CI, if any.
+
+## Wording
+
+- Avoid unnecessarily wordy comments. Be concise.
+- Be neutrally observational; avoiding charged statements and blanket statements
+- Make the recommendation explicit.
+- When possible, put the recommendation first, then the explanation.
+- Avoid "me", "I", "we", etc, since the bot is not a person.
+- Avoid "you" and "your code", because it can sometimes come across as judging 
+  the person rather than the code.
+- Avoid words like "just", "simply", "obviously", which can come across as 
+  condescending: if the comment is necessary, it was _obviously_ not _obvious_.
+- Avoid making assumptions about what people know and don't know. The only
+  exception is knowledge from earlier tracks in the _core_ exercises only. Avoid
+  "as you know", "as you remember", "as you learned", "now that we all
+  undertstand x", because even though something was said, the person doesn't 
+  necessarily understood it.
+
+### Early exercises
```

Comment:
Maybe rename to `### Guidance`?

Reply:
Yes, I like.

Reply:
But It ís especially for early exercises. How would we like to communicate that?

Reply:
I think the sentence is fine.